starboard/extension/accessibility.h

```diff
@@ -0,0 +1,241 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Module Overview: Starboard Accessibility module
+//
+// Provides access to the system options and settings related to accessibility.
+
+#ifndef STARBOARD_EXTENSION_ACCESSIBILITY_H_
+#define STARBOARD_EXTENSION_ACCESSIBILITY_H_
+
+#include "starboard/export.h"
+#include "starboard/types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define kCobaltExtensionAccessibilityName "dev.cobalt.extension.Accessibility"
```

Comment:
@kaidokert we want to use "Starboard extension" instead of "Cobalt extension" for new extensions, right? I think this should be:

`#define kStarboaradExtensionAccessibilityName "dev.starboard.extension.Accessibility"`

Reply:
Correct, yes. It's now [Starboard extensions](https://developers.google.com/youtube/cobalt/docs/codelabs/starboard_extensions/codelab)

Reply:
Renamed